code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -213,9 +213,19 @@ meteor_act
 
 	if(user == src) // Attacking yourself can't miss
 		return target_zone
-
-	var/hit_zone = get_zone_with_miss_chance(target_zone, src)
-
+	var/hit_zone = check_zone(target_zone)
+	//check if we hit
+	var/dodge = src.stats.getStat(STAT_ROB)
+	var/miss_chance = ((base_miss_chance[hit_zone] / 4) + (20 / (1 + 100 / (dodge + 1)) + 10)) //soft cap, minumum 10 at rob 0, 16.5 at 50, 20 at 100, 22 at 150,
```

Comment:
I think altogether swapping the addition with a multiplication would be better (base chance * (1+softcapped dodge), with the value tweaked accordingly for a similar final result, but works well eitherways in terms of balance.

Reply:
This would allowing removing the /4, satisfying Sugarbone's criteria.
It can be further balanced down by decreasing the "1" into 0.9, decreasing the default while allowing it to keep its strong scale with the Head.

Reply:
just gimme the formula, my head hurts when think

Reply:
Don't, make him think